crates/fluvio/src/fluvio.rs

```diff
@@ -145,6 +137,22 @@ impl Fluvio {
         }
     }
 
+    /// Creates a new Fluvio client with the given connector and configuration
+    pub async fn connect_with_connector(
+        connector: DomainConnector,
+        fluvio_config: &FluvioConfig,
```

Comment:
let's not expose ClientConfig

Reply:
make connect_with_client_config pub(crate)?

Reply:
Yes

Reply:
done